instructions.html

```diff
@@ -13,7 +13,7 @@
             <section id="instructions" class="visible" data-title="Test Instructions">
                 <a href="./" class="logo">
                     <img srcset="resources/logo@2x.png 2x" src="resources/logo.png" alt="Speedometer" />
-                    <div class="version">3.0</div>
+                    <div class="version next">next</div>
                 </a>
                 <div class="section-grid">
                     <h1 class="section-header">Speedometer 3 General Instructions</h1>
```

Comment:
Maybe we can remove the version at all here?

Reply:
I've tried now both, kinda prefer having the version consistently displayed in all views on the side.
But I've removed the "3" in the instruction text now.

Reply:
yeah my comment was targeting line 19